system/Router/AutoRouterImproved.php

```diff
@@ -519,7 +519,15 @@ private function checkUriForMethod(string $method): void
             return;
         }
 
-        if (! in_array($method, get_class_methods($this->controller), true)) {
+        // If `getSomeMethod()` exists, only `controller/some-method` should be
+        // accessible. But if a visitor navigates to `controller/somemethod`,
+        // `getSomemethod()` will be checked, and method_exists() will return true.
```

Comment:
```suggestion
        // `getSomeMethod()` will be checked, and method_exists() will return true.
```

Reply:
Auto Routing Improved tries to convert a URI to a controller class/method.

If `controller/somemethod` comes, it will try to check `Controller::getSomemethod()` (not `Controller::getSomeMethod()`). Because it adds prefix `get` and camelizes the URI segment for method.

And in this exaple, there is only `Controller::getSomeMethod()`, but `method_exists()` returns `true` because method names in PHP are case-insensitive.

Reply:
Improved the comment.

Reply:
Thank you for the explanation.